website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -193,6 +193,18 @@ This is a potential attack surface area that you wouldn’t want to be the root
 
 With Unleash, you can architect your feature flagging solution in a way that all user data stays within your applications. When using server-side SDKs, user data remains within your application and is never shared with the Unleash server regardless of your setup. For client-side SDKs, you can retain all user data within your applications by either [self-hosting Unleash](/understanding-unleash/proxy-hosting#you-host-everything), or [self-hosting Unleash Edge](/understanding-unleash/proxy-hosting#unleash-hosts-the-api-you-host-edge). Read more on the [Unleash architecture here](/understanding-unleash/unleash-overview).
 
+### Use Unleash Edge for enterprise-grade data privacy
+
+To take data privacy, scalability, high performance, and resiliency to the next level for large enterprises, we built [Unleash Edge](/reference/unleash-edge), a lightweight proxy layer between your Unleash API and SDKs. You can deploy Unleash Edge within your own infrastructure so PII and sensitive context data never leave your network boundary.
+
+Unleash Edge operates as a local evaluation point for feature flags, meaning that the evaluation of flags and user context data happens within your infrastructure. It takes a copy of your feature flag data locally and by default, it uses an in-memory cache to store the features it fetches from your Unleash instance. Another benefit to Edge functioning as a local evaluation point is preventing performance issues in one part of your system from affecting others and maintaining service level agreements required by compliance frameworks.
```

Comment:
What is the 'SLAs required by compliance frameworks' referring to? It's not something I've come across before